propkatraj/tests/test_regression.py

```diff
@@ -52,46 +52,19 @@ def pka_from_file(filename):
     return resnums, pkas
 
 
-@pytest.mark.parametrize('selection', ['protein', 'array', 'list'])
+@pytest.mark.parametrize('selection', ['default', 'protein'])
 def test_single_frame_regression(tmpdir, u, selection):
-    """Single frame propkatraj call compared against same frame written by
-    MDA
-    """
-    with tmpdir.as_cwd():
-        if selection == 'array':
-            selection = u.select_atoms('protein').ix
-        elif selection == 'list':
-            selection = u.select_atoms('protein').ix.tolist()
-
-        pkas = propkatraj.get_propka(u, sel=selection, stop=1)
-
-        # load reference data
-        ref_resnums, ref_pkas = pka_from_file(PSF_FRAME_ZERO_PKA)
-
-        # test residue numbers
-        resnums = pkas.columns.to_numpy()
-        assert_equal(resnums, ref_resnums)
-
-        # test pka values
-        assert_almost_equal(pkas.values[0], ref_pkas, decimal=2)
-
-
-@pytest.mark.parametrize('selection', ['protein', 'ag'])
-def test_single_frame_regression_analysisbase(tmpdir, u, selection):
     """Single frame PropkaTraj regression test"""
     with tmpdir.as_cwd():
-        if selection == 'ag':
-            pkatraj = propkatraj.PropkaTraj(u.select_atoms('protein'),
+        if selection != 'default':
```

Comment:
I can understand removing array since it wasn't in the parametrize call, but I don't necessairly  understand the need to change the rest. Maybe I'm missing the value it provides?

Reply:
@IAlibay This might be better.  This way we can feed in new selections easily as long as they aren't `'default'` or `'ag'`.

Reply:
So one way to re-phrase my comment is: "are we considering a case where we might have some other way of selecting 'protein' that isn't the approach used here"?

Any other regression test would fall within a different test. Hence my inability to be convinced about the necessity of further changes here, unless there's a selection method I'm not considering?

Reply:
Oh, I see. If there is no desire to expand the tests to other selections then this code doesn't add anything new. I can revert it.